source/Engine.cpp

```diff
@@ -2322,15 +2331,31 @@ void Engine::AddSprites(const Ship &ship)
 {
 	bool hasFighters = ship.PositionFighters();
 	double cloak = ship.Cloaking();
+	bool damageHighlight = Preferences::Has("Damage highlights");
 	bool drawCloaked = (cloak && ship.IsYours());
+	bool drawHeat = ship.Heat() > 0.9;
+	bool drawShield = ship.RecentShield() > 4.;
+	double shield = sqrt(ship.RecentShield()/ship.Attributes().Get("shields"));
+	double heat = min((ship.Heat()-0.9) / 16, 1.);
 	auto &itemsToDraw = draw[calcTickTock];
-	auto drawObject = [&itemsToDraw, cloak, drawCloaked](const Body &body) -> void
+	auto drawObject = [&itemsToDraw, cloak, shield, heat, drawCloaked, drawShield, drawHeat, damageHighlight](const Body &body) -> void
 	{
-		// Draw cloaked/cloaking sprites swizzled red, and overlay this solid
-		// sprite with an increasingly transparent "regular" sprite.
+		// Draw cloaked/cloaking sprites swizzled transparent grey.
 		if(drawCloaked)
-			itemsToDraw.AddSwizzled(body, 27);
-		itemsToDraw.Add(body, cloak);
+			itemsToDraw.AddSwizzled(body, 10, 0.7);
```

Comment:
Not sure whats going on with the drawing changes and if they're necessary since I dont know that much about that so would be grateful it smbd that does could have a look.
You do realize drawCloaked was to show only player ships that are cloaked in red, and not enemy ships?

Whatever the case you need to adjust the indentation

Reply:
Probably better we don't have enemy ships outlined while cloaking, that's not so much visual clarification as it is almost cheating by outlining them more. The idea is to just do it for player ships.